# Destination address
string destination

# Destination port
uint16 destination_port

# Source port
uint16 source_port
